bluesky/preprocessors.py

```diff
@@ -352,6 +352,42 @@ def _inner_plan():
                                         _unsubscribe()))
 
 
+def suspend_wrapper(plan, suspenders):
+    """
+    Install suspenders to the RunEngine, and remove them at the end.
+
+    Parameters
+    ----------
+    plan : iterable or iterator
+        a generator, list, or similar containing `Msg` objects
+    suspenders : suspender or list of suspenders
+        Suspenders to use for the duration of the wrapper
+
+    Yields
+    ------
+    msg : Msg
+        messages from plan, with 'install_suspender' and 'remove_suspender'
+        messages inserted and appended
+    """
+    if not isinstance(suspenders, list):
```

Comment:
Can you use `collections.Iterable` instead of `list` so that a `tuple` (etc.) is also accepted?